src/OpenColorIO/transforms/builtins/Displays.cpp

```diff
@@ -201,6 +201,50 @@ void RegisterAll(BuiltinTransformRegistryImpl & registry) noexcept
                             CIE_XYZ_D65_to_P3_D60_BFD_Functor);
     }
 
+    {
+        auto CIE_XYZ_D65_to_DCDM_D65_Functor = [](OpRcPtrVec & ops)
+        {
+            const double scale     = 48.0 / 52.37;
+            const double scale4[4] = { scale, scale, scale, 1. };
+            CreateScaleOp(ops, scale4, TRANSFORM_DIR_FORWARD);
+
+            const GammaOpData::Params rgbParams   = { 2.6 };
+            const GammaOpData::Params alphaParams = { 1.0 };
+            auto gammaData = std::make_shared<GammaOpData>(GammaOpData::BASIC_REV,
+                                                           rgbParams, rgbParams, rgbParams, alphaParams);
+            CreateGammaOp(ops, gammaData, TRANSFORM_DIR_FORWARD);
+        };
+
+        registry.addBuiltin("DISPLAY - CIE-XYZ-D65_to_DCDM-D65", 
+                            "Convert CIE XYZ (D65 white) to Gamma 2.6, XYZ-E with D65 white",
+                            CIE_XYZ_D65_to_DCDM_D65_Functor);
+    }
+
+    {
+        auto CIE_XYZ_D65_to_DCDM_D60_BFD_Functor = [](OpRcPtrVec & ops)
+        {
+            const MatrixOpData::Offsets d65_wht_XYZ(0.95045592705167, 1., 1.08905775075988, 0.);
+            const MatrixOpData::Offsets d60_wht_XYZ(0.95264607456985, 1., 1.00882518435159, 0.);
+            MatrixOpData::MatrixArrayPtr matrix
+                = build_vonkries_adapt(d65_wht_XYZ, d60_wht_XYZ, ADAPTATION_BRADFORD);
+            CreateMatrixOp(ops, matrix, TRANSFORM_DIR_FORWARD);
+
+            const double scale     = 48.0 / 52.37;
+            const double scale4[4] = { scale, scale, scale, 1. };
+            CreateScaleOp(ops, scale4, TRANSFORM_DIR_FORWARD);
+
+            const GammaOpData::Params rgbParams   = { 2.6 };
+            const GammaOpData::Params alphaParams = { 1.0 };
+            auto gammaData = std::make_shared<GammaOpData>(GammaOpData::BASIC_REV,
+                                                           rgbParams, rgbParams, rgbParams, alphaParams);
+            CreateGammaOp(ops, gammaData, TRANSFORM_DIR_FORWARD);
+        };
+
+        registry.addBuiltin("DISPLAY - CIE-XYZ-D65_to_DCDM-D60-BFD", 
```

Comment:
Is this display intended to be used with the ACES2 Builtins (same for the HDR variant)? Note that there is no [CAT](https://github.com/remia/OpenColorIO/blob/aces2-output-transforms/src/OpenColorIO/transforms/builtins/ACES.cpp#L562) to D65 performed at the moment in the ACES2 builtins, which I believe was handled similarly in ACES 1.x D60 transforms. Please let me know if I need to update something on the ACES2 PR side.

Reply:
Yes, it's intended to be used for the ACES 2 config. But I think what you're proposing is that you would use the "DISPLAY - CIE-XYZ-D65_to_DCDM-D65" for both the D65 and D60 view transforms. The D60 version will essentially have a "D60 sim" type of skew to it. I will remove the D60 versions.